packages/studio-base/webpack.ts

```diff
@@ -37,6 +37,8 @@ type Options = {
   allowUnusedVariables?: boolean;
   /** Specify the app version. */
   version: string;
+  /** Specify the Storybook config path for Type checking */
+  storybookConfigPath?: string;
```

Comment:
I think calling out that this was explicitly added for Storybook is okay, but I don't see a strong reason not to expose this as a non-storybook-specific option to makeConfig.
```suggestion
  /** Specify the path to the tsconfig.json file for ForkTsCheckerWebpackPlugin. If unset, the plugin defaults to finding the config file in the webpack `context` directory. */
  tsconfigPath?: string;
```